src/logging_vnode.erl

```diff
@@ -779,11 +779,13 @@ handle_commit(TxId, OpPayload, T, Key, MinSnapshotTime, MaxSnapshotTime, Ops, Co
         filter_terms_for_key(T, Key, MinSnapshotTime, MaxSnapshotTime, Ops, CommittedOpsDict)
     end.
 
-check_min_time(SnapshotTime, MinSnapshotTime) ->
-    ((MinSnapshotTime == undefined) orelse (vectorclock:ge(SnapshotTime, MinSnapshotTime))).
+-spec check_min_time(snapshot_time(), snapshot_time() | undefined) -> boolean().
+check_min_time(_SnapshotTime, undefined) -> true;
+check_min_time(SnapshotTime, MinSnapshotTime) -> vectorclock:ge(SnapshotTime, MinSnapshotTime).
 
-check_max_time(SnapshotTime, MaxSnapshotTime) ->
-    ((MaxSnapshotTime == undefined) orelse (vectorclock:le(SnapshotTime, MaxSnapshotTime))).
+-spec check_max_time(snapshot_time(), snapshot_time() | undefined) -> boolean().
+check_max_time(_SnapshotTime, undefined) -> true;
+check_max_time(SnapshotTime, MaxSnapshotTime) -> vectorclock:le(SnapshotTime, MaxSnapshotTime).
```

Comment:
Slight coverage decrease (one line) but looks a bit nicer and more readable in my opinion